compose-files/brizo.yml

```diff
@@ -19,5 +19,8 @@ services:
       SECRET_STORE_URL: ${SECRET_STORE_URL}
       PARITY_ADDRESS: ${PUBLISHER_ADDRESS}
       PARITY_PASSWORD: ${PUBLISHER_PASSWORD}
+      PARITY_KEYFILE: ${PUBLISHER_KEYFILE}
+      LOG_LEVEL: ${BRIZO_LOG_LEVEL}
     volumes:
       - ${KEEPER_ARTIFACTS_FOLDER}:/usr/local/keeper-contracts:ro
+      - ../networks/spree/authorities/:/parity-data/keys
```

Comment:
i think it must be ${KEEPER_NETWORK} instead of spree, or it will always use spree